quickstarts/microsoft.sql/sql-data-warehouse-transparent-encryption-create/azuredeploy.json

```diff
@@ -38,11 +38,19 @@
         "description": "Enable/Disable Transparent Data Encryption"
       }
     },
-    "serviceLevelObjective": {
+    "capacity": {
+      "type": "int",
+      "metadata": {
+        "description": "The Elastic Pool DTU or nomber of vcore."
```

Comment:
@cmendible Please review this. This should not be labeled as an elastic pool DTU or number of vcores, but rather a string that is the DW size, for example, DW400c.

Reply:
Fixed based on the values provided in this table: https://docs.microsoft.com/en-us/azure/synapse-analytics/sql-data-warehouse/sql-data-warehouse-service-capacity-limits#workload-management and the description displayed in the Azure portal.